src/pages/Markets/components/LpCard.tsx

```diff
@@ -32,7 +32,14 @@ export const LpCard: React.FC<LpCardProps> = ({ assetId, apy, volume24H, onClick
   }, [volume24H])
 
   const apyValue = useMemo(() => {
-    return <Amount.Percent autoColor value={bnOrZero(apy).times(0.01).toString()} />
+    const isBiggerThan1000 = bnOrZero(apy).gt(1000)
+    return (
+      <Amount.Percent
+        autoColor
+        value={isBiggerThan1000 ? '10' : bnOrZero(apy).times(0.01).toString()}
+        suffix={isBiggerThan1000 ? '+' : undefined}
```

Comment:
preferably-blocking: this gets a bit messy view-layer wise and the cognitive load on this 10 being 1000% and the inline boilerplate is quite high. See https://github.com/shapeshift/web/pull/9242 for something a bit cleaner 🙏🏽